### PR TITLE
Address static analysis issues in SSH remoting

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
- remoteRunspaces is not used TFS:10618323
- not checking return value of ResumeThread() TFS:10618344

### CODE REVIEW

File: src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs [hunk 1]
```diff
@@ -2288,7 +2288,12 @@ private string GetCurrentUserName()
 
                 // At this point, we should have a suspended process.  Get the .Net Process object, resume the process, and return.
                 Process result = Process.GetProcessById(lpProcessInformation.dwProcessId);
-                PlatformInvokes.ResumeThread(lpProcessInformation.hThread);
+                uint returnValue = PlatformInvokes.ResumeThread(lpProcessInformation.hThread);
+
+                if (returnValue == 4294967295) // same as (DWORD)-1
```
Comment (mirichmo) [inline src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs]: We should not use magic numbers in the code. Please define a descriptive constant for this value.

**Review by mirichmo — CHANGES_REQUESTED** [review]

File: src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs (hunk 1 above)
Comment (SteveL-MSFT) [inline src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs]: fixed

**Comment by SteveL-MSFT** [comment]
@mirichmo fixed

**Review by PaulHigin — APPROVED** [review]
LGTM

File: src/System.Management.Automation/utils/PlatformInvokes.cs [hunk 2]
```diff
@@ -701,6 +701,7 @@ public SECURITY_ATTRIBUTES()
 
         [DllImport(PinvokeDllNames.ResumeThreadDllName, CharSet = CharSet.Unicode, SetLastError = true)]
         public static extern uint ResumeThread(IntPtr threadHandle);
+        internal static uint RESUME_THREAD_FAILED = 4294967295; // (DWORD)-1
```
Comment (mirichmo) [inline src/System.Management.Automation/utils/PlatformInvokes.cs]: Please use System.UInt32.MaxValue instead

**Review by mirichmo — CHANGES_REQUESTED** [review]

File: src/System.Management.Automation/utils/PlatformInvokes.cs (hunk 2 above)
Comment (SteveL-MSFT) [inline src/System.Management.Automation/utils/PlatformInvokes.cs]: will fix, seems obvious now :)

**Review by mirichmo — APPROVED** [review]

**Pull request merged**